app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/MainScaffold.kt

```diff
@@ -65,6 +72,16 @@ private fun MainScaffold(
     currentDestination: MainNavigation,
     navController: NavHostController,
     content: @Composable () -> Unit,
+) {
+    Navigation(currentDestination, navController::navigateToSelectedItem, navType, content)
+}
+
+@Composable
+private fun Navigation(
```

Comment:
You can just directly update `MainScaffold` and add this callback instead of another method